modules/apps/27-interchain-accounts/controller/keeper/keeper.go

```diff
@@ -144,11 +154,15 @@ func (k Keeper) GetAllActiveChannels(ctx sdk.Context) []genesistypes.ActiveChann
 	for ; iterator.Valid(); iterator.Next() {
 		keySplit := strings.Split(string(iterator.Key()), "/")
 
+		portID := keySplit[1]
+		connectionID := keySplit[2]
+		channelID := string(iterator.Value())
```

Comment:
extracted these out to make it more clear which values are being passed into `IsMiddlewareEnabled`